src/dependabot/output.ts

```diff
@@ -20,16 +20,27 @@ export function set (updatedDependencies: Array<updatedDependency>): void {
   const dependencyType = maxDependencyTypes(updatedDependencies)
   const updateType = maxSemver(updatedDependencies)
 
+  const firstDependency = updatedDependencies[0]
+  const directory = firstDependency?.directory
+  const ecosystem = firstDependency?.packageEcosystem
+  const target = firstDependency?.targetBranch
+
```

Comment:
The `directory`, `ecosystem`, and `target branch` should be the same across an update job, so this should be okay 👌🏾